apps/src/calc/calc.js

```diff
@@ -143,16 +154,9 @@ Calc.init = function(config) {
         config.forceInsertTopBlock);
     }
 
-    appState.targetExpressions = generateExpressionsFromBlockXml(solutionBlocks);
+    appState.targetSet = generateEquationSetFromBlockXml(solutionBlocks);
```

Comment:
Ah, got it. Would `targetEquations` fit the bill? What feels most sorely missing is the `equation` relationship from this context.

nbd either way, just a nit suggestion